metadata-service/plugin/src/main/java/com/datahub/plugins/common/PluginConfigWithJar.java

```diff
@@ -0,0 +1,10 @@
+package com.datahub.plugins.common;
+
+import java.nio.file.Path;
+
+
+public interface PluginConfigWithJar extends PluginConfig {
```

Comment:
Minor: It feels like we have too many classes and abstractions floating around in here. Are they really _all_ necessary or can we simplify?

Reply:
Simplified